instance_template/main.tf

```diff
@@ -0,0 +1,335 @@
+/**
```

Comment:
As per the [Standard Module Structure article](https://www.terraform.io/docs/modules/create.html#standard-module-structure), nested modules must be moved under the `modules` directory.